src/components/LeftPanelWidget.tsx

```diff
@@ -44,7 +44,7 @@ export interface IVolumeMetadata {
     mount_point: string,
     size?: string,
     size_type?: string,
-    annotation?: string,
+    annotation?: {key: string, value: string},
```

Comment:
This would currently break the backend when Kale reads in the notebook metadata, as it expects a string in the form of `key:value`

Reply:
I've fixed that on kubeflow-kale/kale#8

Reply:
Next releases should be cut containing these two PRs (one in each component)

Reply:
oh yeah just checking that one